rust/src/mask/config/mod.rs

```diff
@@ -168,9 +176,20 @@ pub struct MaskConfig {
 
 impl MaskConfig {
     /// Returns the number of bytes needed for an element of a mask object.
+    ///
+    /// # Panics
+    /// Panics if the bytes per number can't be represented as usize.
     pub(crate) fn bytes_per_number(&self) -> usize {
         let max_number = self.order() - BigUint::from(1_u8);
-        (max_number.bits() + 7) / 8
+        let bpn = (max_number.bits() + 7) / 8;
+        if bpn > MAX_BPN {
+            // the largest bpn from the masking configuration catalogue is currently 173, hence
+            // this is almost impossible on 32 bits targets and smaller targets are currently not
+            // of interest for us
+            unimplemented!("the employed masking config is not supported on the target")
```

Comment:
I think `panic!` is more appropriate here, since it _cannot_ ever be implemented.

Reply:
changed. i would guess it is possible to implement this with a larger fixed size type than `usize` and a lot of ugly memory juggling, but i get your point.